ports/comms-ublox/portfile.cmake

```
@@ -3,28 +3,23 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO commschamp/cc.ublox.generated
-    REF v0.20.2
-    SHA512 5672d964ea3e505837e44a5fd928069a219a5731764cb54bfe8609e39c6c6dd0059660bcde317c6c60cd1bd8d1f7942d2faa022095bf651817568291bc6a7569
+    REF v1.0
+    SHA512 0c487d9409c2f2818024f6232832762527250c3563a5eb5c639ad49943931ceb24616db2432bcd752d1a84820ec5349522510dcd202508641d3f29aef41ca1e5
     HEAD_REF master
-    PATCHES
-        fix-comms.patch
 )
 
-vcpkg_configure_cmake(
+vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
-    PREFER_NINJA
     OPTIONS
-        -DOPT_BUILD_TEST=OFF
-        -DOPT_BUILD_PLUGIN=OFF
-        -DOPT_NO_COMMS=ON
-        -DOPT_EXTERNALS_UPDATE_DISCONNECTED=ON
+        -DOPT_REQUIRE_COMMS_LIB=ON
+        -DCMAKE_DISABLE_FIND_PACKAGE_Doxygen=OFF
 )
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 
-vcpkg_fixup_cmake_targets(CONFIG_PATH lib/ublox/cmake TARGET_PATH share/ublox)
+vcpkg_cmake_config_fixup(PACKAGE_NAME ublox CONFIG_PATH lib/ublox/cmake)
 # currently this is only a header only library. after moving lib/ublox to share this lib path will be empty
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib")
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug")
 
 # Handle copyright
-file(INSTALL "${SOURCE_PATH}/LICENSE.txt" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+file(INSTALL "${CURRENT_PORT_DIR}/copyright" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)

```

```suggestion
vcpkg_download_distfile(
    license
    URLS "https://raw.githubusercontent.com/commschamp/commsdsl/e0f0bf42a455445d0a266d3540b008e0dc24cd15/LICENSE.txt"
    FILENAME "commsdsl-e0f0bf42a-license.txt"
    SHA512 0
)
configure_file("${license}" "${CURRENT_PACKAGES_DIR}/share/${PORT}/copyright" COPYONLY)
```

and please delete `copyright`.

---

As I have previously mentioned, the downloaded license would not be correct for the sources. So I'm not in favor for the change. But we could force the user into the apache license and if they want another license, they would have to download or generate the source by themselves.

Your final decision.